spec/api/v1/patients_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'rails_helper'
+require 'active_record'
+# rubocop:disable RSpec/MultipleExpectations
+RSpec.describe 'patient', type: :request do
+  let(:patients_crud_url) { '/api/v1/patients' }
+  let(:patients_params) do
+    {
+      address: Faker::Address.full_address,
+      date_of_birth: Faker::Date.between(from: '1900-01-01', to: Time.zone.today),
+      email: Faker::Internet.email,
+      full_name: Faker::Name.name,
+      notes: 'Something notes',
+      telephone_number: Faker::Number.number(digits: 7),
+      passport_id: Faker::Alphanumeric.alpha(number: rand(6..50)),
+      allergies_additional: 'Something information about allergies',
+      allergies: ''
+    }
+  end
+
+  before do
+    create_list :patient, 5
+  end
+
+  after do
+    Patient.destroy_all
```

Comment:
@alhimick-prog can you describe why do we need it there?

Reply:
It's not needed. It was an unnecessary precaution for the cleanliness of the table.